code/modules/reagents/Chemistry-Machinery.dm

```
@@ -101,7 +101,9 @@
 
 	var chemicals[0]
 	for (var/re in dispensable_reagents)
-		var/datum/reagent/temp = chemical_reagents_list[re]
+		var/datum/reagent/temp
+		if(re in chemical_reagents_list)
+			temp = chemical_reagents_list[re]

```

You'd rather it fail silently if someone fucks up? That's retarded.